homework_03/05_Viktoriya_Vasileva/test_Circles.py

```diff
@@ -0,0 +1,74 @@
+from Circles import Point, Circle, RelativePosition
+
+
+def test_outside_circles():
+    circle_a = Circle(Point(1.0, 1.0), 1.0)
+    circle_b = Circle(Point(10.0, 10.0), 2.0)
+    assert circle_a.find_relative_position(circle_b) == RelativePosition.NO_COMMON_POINTS
+    
+    circle_a = Circle(Point(1.0, 1.0), 1.0)
+    circle_b = Circle(Point(10.0, 1.0), 7.0)
+    assert circle_a.find_relative_position(circle_b) == RelativePosition.NO_COMMON_POINTS
+
+    circle_a = Circle(Point(1.0, 1.0), 1.0)
+    circle_b = Circle(Point(10.0, 1.0), 7.99999)
+    assert circle_a.find_relative_position(circle_b) == RelativePosition.NO_COMMON_POINTS
```

Comment:
Because of the rounding, this assert tests wrong behaviour and accepts it as the right one. The circle on lines 13 and 14 are TOUCHING, you can check it here: https://planetcalc.com/8098/